express_server.js

```diff
@@ -16,8 +16,14 @@ function generateRandomString() {
 }
 
 const urlDatabase = {
-  "b2xVn2": "http://www.lighthouselabs.ca",
-  "9sm5xK": "http://www.google.com"
+  "b2xVn2": {
```

Comment:
```suggestion
  b2xVn2: {
```
since this is a js object, do u really need those apostrophes ?